prow/plugins/approve/approvers/owners.go

```diff
@@ -633,8 +641,13 @@ Approval requirements bypassed by manually added approval.
 This pull-request has been approved by:{{range $index, $approval := .ap.ListApprovals}}{{if $index}}, {{else}} {{end}}{{$approval}}{{end}}
 
 {{- if (and (not .ap.AreFilesApproved) (not (call .ap.ManuallyApproved))) }}
-To complete the [pull request process]({{ .prProcessLink }}), please assign {{range $index, $cc := .ap.GetCCs}}{{if $index}}, {{end}}**{{$cc}}**{{end}} after the PR has been reviewed.
-You can assign the PR to them by writing `+"`/assign {{range $index, $cc := .ap.GetCCs}}{{if $index}} {{end}}@{{$cc}}{{end}}`"+` in a comment when ready.
+{{ if len .ap.SuggestedCCs -}}
+To complete the [pull request process]({{ .prProcessLink }}), please assign {{range $index, $cc := .ap.SuggestedCCs}}{{if $index}}, {{end}}**{{$cc}}**{{end}} after the PR has been reviewed.
+You can assign the PR to them by writing `+"`/assign {{range $index, $cc := .ap.SuggestedCCs}}{{if $index}} {{end}}@{{$cc}}{{end}}`"+` in a comment when ready.
+{{- end}}
+{{- if len .ap.AssignedCCs -}}
```

Comment:
If `GetCCs()` returns multiple users and some are assignees while others are not, we'll output both of these sections which seems wrong to me. Please add a unit test for this case too.

Reply:
It was on purpose... What do you suggest then?

Reply:
Correction: I meant confusing not wrong. I think we **do** want to recommend both assigning the SuggestedCCs and requesting approval from the AssignedCCs, but I think it is confusing to have two lines with different instructions for "To complete the pull request process...".

To me it would be clearer to have one set of instructions when we have both suggested and assigned CCs. Something like this:
> To complete the pull request process, please ask for approval from cjwagner and additionally assign mattyx after the PR has been reviewed.
You can assign the PR to them by writing `/assign matthyx` in a comment when ready

Reply:
done this part for review, although I need to add some test cases suggested by @chaodaiG